From d05d4196886b58204904d0d43a1052689a1f26a8 Mon Sep 17 00:00:00 2001
From: Tsuyoshi Ozawa <ozawa@apache.org>
Date: Wed, 22 Apr 2015 05:47:59 +0900
Subject: [PATCH 1143/2748] YARN-3495. Confusing log generated by
 FairScheduler. Contributed by Brahma Reddy
 Battula.

(cherry picked from commit 105afd54779852c518b978101f23526143e234a5)
(cherry picked from commit 0b87ae27aadff1b16f95c5b61b53d597d2377dcc)

Change-Id: I55c97488d5c98448125df322dc504bbc41446d77
---
 .../scheduler/fair/FairScheduler.java              |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FairScheduler.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FairScheduler.java
index 6a02ef8..f6124bf 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FairScheduler.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FairScheduler.java
@@ -818,7 +818,8 @@ private synchronized void removeApplicationAttempt(
   protected synchronized void completedContainer(RMContainer rmContainer,
       ContainerStatus containerStatus, RMContainerEventType event) {
     if (rmContainer == null) {
-      LOG.info("Null container completed...");
+      LOG.info("Container " + containerStatus.getContainerId()
+          + " completed with event " + event);
       return;
     }
 
@@ -831,7 +832,7 @@ protected synchronized void completedContainer(RMContainer rmContainer,
         container.getId().getApplicationAttemptId().getApplicationId();
     if (application == null) {
       LOG.info("Container " + container + " of" +
-          " unknown application attempt " + appId +
+          " finished application " + appId +
           " completed with event " + event);
       return;
     }
-- 
1.7.9.5

